contracts/injective-cosmwasm-mock/src/msg.rs

```diff
@@ -0,0 +1,19 @@
+use cosmwasm_std::Coin;
+use injective_cosmwasm::{MarketId, SubaccountId};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct InstantiateMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    TestDepositMsg { subaccount_id: SubaccountId, amount: Coin },
```

Comment:
This is only the first example message I added. Obviously all the other messages within `injective-cosmwasm` are still missing.

Reply:
Yes, I'll add the other in a new PR